task09/task09.4/task09.4/Program.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace task09._4
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            for (int i = 10000; i > 0; i--)
+            {
+                int m = 1, sum = default;
+                while (m != i)
+                {
+                    if (i % m == 0)  sum += m;
+                    m++;
+                }
+                if (sum == i)
+                {
+                    Console.WriteLine(i);
+                }
```

Comment:
Когда внутри` if` только одна строка, то `{ }` можно не писать.
```suggestion
                if (sum == i)
                    Console.WriteLine(i);
```